manifests/load.pp

```diff
@@ -18,7 +18,15 @@
 
   case $ensure {
     'present': {
-      $changes = "clear '${name}'"
+      case $::osfamily {
+        'Debian': {
+          $changes = "clear '${name}'"
+        }
+        'Suse': {
+          $changes = "set MODULES_LOADED_ON_BOOT/value[.='${name}'] '${name}'"
+        }
+        default: { }
```

Comment:
The $changes is set for Augeas. RedHat uses a template file, and therefore has no need for that $changes variable at all. See the whole file, at the bottom.
The default: case is empty, to please puppet-lint. If wanted, I could add a comment into that default: statement, to make it more clear.

same applies to the similar snipped in the 'absent' case below.